systems_cross_building/windows_subsystems.rst

```diff
@@ -37,6 +37,30 @@ shouldn't specify it.
 Running commands inside the subsystem
 -------------------------------------
 
+
+.. _conanfile_win_bash:
+
+self.win_bash (New in 1.39)
+___________________________
```

Comment:
I wouldn't add the version mark in the title as when we remove that in the future it might break URL references

```suggestion
self.win_bash
_____________
```